### PR TITLE
Change OS X to macOS

### PR DESCRIPTION
**Pull request by samdoran**

### CODE REVIEW

**Comment by samdoran** [comment]
I updated the CHANGELOG.

File: mackup/main.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 """Mackup.
 
 Keep your application settings in sync.
-Copyright (C) 2013-2015 Laurent Raufaste <http://glop.org/>
+Copyright (C) 2013-2018 Laurent Raufaste <http://glop.org/>
```
Comment (treee111) [inline mackup/main.py:4]: could be updated to 2021 nowadays

**Review by treee111 — APPROVED** [review]
A good addition to keep up with the naming of Mac OS!

**Review by lra — APPROVED** [review]

File: mackup/main.py (hunk 1 above)
Comment (lra) [reply mackup/main.py]: https://github.com/lra/mackup/pull/1023/commits/b88f6caf0cb5c7d46dae5390b4a623edaa4296a4

**Comment by samdoran** [comment]
Thank you for merging this.

**Pull request merged**